javaslang-benchmark/src/test/java/javaslang/benchmark/MicroBenchmark.java

```diff
@@ -1,30 +1,19 @@
 package javaslang.benchmark;
 
-import javaslang.collection.List;
-import javaslang.collection.Queue;
-import javaslang.collection.Stream;
-import javaslang.collection.Vector;
+import javaslang.collection.*;
 import org.junit.Test;
 import org.openjdk.jmh.annotations.*;
-import org.openjdk.jmh.infra.Blackhole;
 import org.openjdk.jmh.runner.Runner;
-import org.openjdk.jmh.runner.options.Options;
-import org.openjdk.jmh.runner.options.OptionsBuilder;
-import org.openjdk.jmh.runner.options.TimeValue;
+import org.openjdk.jmh.runner.options.*;
 
-
-import java.util.Random;
 import java.util.concurrent.TimeUnit;
 
 @State(Scope.Benchmark)
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 @BenchmarkMode(Mode.Throughput)
 public class MicroBenchmark {
```

Comment:
Would it be worth adding benchmark tests for the java.util.PriorityQueue counterpart methods as well? In that way we will be able to see at a glance how fast/slow our implementation is in relation with the existing java.util library?

I have done this for the TreeMapBenchmark (still under development) and it works very well.  I have used use method names such as "listHeadSlang" and "listHeadJava" (as opposed to "slangListHead"). In that way the performance report shows related Slang/Java implementations together, and thus easy to compare.

As a matter of fact, I have also been thinking in adding Scala immutable benchmark tests.

Also, I would use parameter "10" instead of "1". We want to cover the common use case of having "a few elements" in the PriorityQueue. 1 is probably too few, while 100 is "a few" too many.